src/main/java/core/basesyntax/FileHelper.java

```diff
@@ -0,0 +1,149 @@
+    private static final String PATH_TO_FILE_DOESNT_EXISTS_MSG = "Path %s doesn't exists. "
+            + "Can't create the file.\n";
+    private static final String REG_EXP_MULTI_WHITESPACE = "[\\s]+";
+    private static final String FILE_INFO_FORMAT_STRING =
+            "File info:\n%20s: %s\n%20s: %s bytes\n%20s: %s\n%20s: %s\n%20s: %s\n";
+    private static final String LAST_MODIFIED_PARAM_TITLE = "last modified";
+    private static final String SIZE_PARAM_TITLE = "size";
+    private static final String NUMBER_OF_STRINGS_PARAM_TITLE = "number of strings";
+    private static final String NUMBER_OF_WORDS_PARAM_TITLE = "number of words";
+    private static final String NUMBER_OF_CHARS_PARAM_TITLE = "number of chars";
+    private static final String FILE_DOESNT_EXIST_ERROR_MSG = "File %s doesn't exist.\n";
+    private static final DateTimeFormatter FORMAT =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
+
+    private static FileHelper instance;
+    private HashMap<Command, String> helpInfo;
+
+    private FileHelper() {
+        helpInfo = new HashMap<>();
+        helpInfo.put(Command.CREATE, CREATE_COMMAND_HELP_INFO);
+        helpInfo.put(Command.READ, READ_COMMAND_HELP_INFO);
+        helpInfo.put(Command.INFO, INFO_COMMAND_HELP_INFO);
+        helpInfo.put(Command.HELP, HELP_COMMAND_HELP_INFO);
+        helpInfo.put(Command.EXIT, EXIT_COMMAND_HELP_INFO);
+    }
+
+    public static FileHelper getInstance() {
+        if (instance == null) {
+            instance = new FileHelper();
+        }
+        return instance;
+    }
+
+    void createFile(String path, String fileName, String fileContent) {
+        File file = new File(path + File.separator + fileName);
+        File parentDir = file.getParentFile();
+        if (!parentDir.exists()) {
+            System.out.format(PATH_TO_FILE_DOESNT_EXISTS_MSG, parentDir.getAbsolutePath());
+            return;
+        }
+        if (file.exists() && ! ConsoleHelper.getInstance()
+                .askYesNoQuestion(REQUEST_FILE_OVERRIDE_QUESTION)) {
+            return;
+        }
+        try {
+            Files.write(file.toPath(), fileContent.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    void showFileInfo(String path, String fileName) {
+        File file = new File(path + File.separator + fileName);
+        if (!file.exists()) {
+            System.out.format(FILE_DOESNT_EXIST_ERROR_MSG, file.getAbsolutePath());
+            return;
+        }
+        int numberOfStrings = 0;
+        int numberOfWords = 0;
+        int numberOfChars = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String str;
+            while ((str = reader.readLine()) != null) {
+                numberOfStrings++;
+                numberOfChars += str.length();
+                numberOfWords += str.split(REG_EXP_MULTI_WHITESPACE).length;
```

Comment:
You can make your method much shorter if use nio Files.readAllLines(file).size() = lines.And you can put it right into your sout. Google method for other cases)